lr_face/utils.py

```diff
@@ -250,3 +250,10 @@ def get_enfsi_lrs():
         df_enfsi = df_enfsi.append(df_temp)
 
     return df_enfsi.replace('-', 0)
+
+
+def get_valid_scores(p, calibration_pairs):
```

Comment:
nice. I guess we also need to handle -1's at test time? That is a separate issue?

Reply:
yes, I will also do that in this PR, but it wasn't done yet